bitrise.yml

```diff
@@ -822,6 +823,33 @@ workflows:
           inputs:
             - dir_to_check: $CLONE_INTO_DIR
 
+  _test_sparse_checkout_commit_on_branch:
```

Comment:
Can you also add test for checking out commit cfba2b01332e31cb1568dbf3f22edce063118bae (the same as in _test_unshallow)?